### PR TITLE
You can no longer convert objective targets to your blood brother team

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

You can no longer convert your objective targets to your blood brother team.

This means the usual "commit violence against this user" objectives, but also includes protect objectives and should (theoretically) prevent you from converting targets from other conflicting objectives such as mutiny/sacrifice/obsession targets.
## Why It's Good For The Game

This kind of fucks up the whole antagonist on a conceptual level. Pick ANYONE else please!
## Changelog
:cl: Rhials
fix: You can no longer convert assassination targets to your blood brother team.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/brother/brother.dm [hunk 1]
```diff
@@ -63,6 +63,11 @@
 		flashed.balloon_alert(source, "[flashed.p_their()] mind is vacant!")
 		return
 
+	for(var/datum/objective/assassinate/assassinate_objective as anything in source.mind.get_all_objectives())
+		if(flashed == assassinate_objective.target.current)
```
Comment (MrMelbert) [inline code/modules/antagonists/brother/brother.dm:67]: This is either 
- An improper use of `as anything` (there will be more than assassination objectives in all objectives of a mob) 

Or
- Something that doesn't even need to be limited to assassination objectives (So it works with all similar objectives, IE Maroon, Mutiny, Sacrifice)

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/brother/brother.dm (hunk 1 above)
Comment (Rhials) [reply code/modules/antagonists/brother/brother.dm]: Didn't occur to me that `target` was set on the objective level. I believe it is the latter.

**Comment by the-orange-cow** [comment]
b-b-but mah metafriend!!!

**Comment by Likteer** [comment]
> b-b-but mah metafriend!!!


So this PR'd be aimed at particular, persistent friends who team up on every occasion?
I think you are oversensitive on that problem


This PR solves the issue of BB flashing & making their target die willingly.
It's about cheesing objectives, or even going solo.
Current BB rewards picking players you know are strong. This will not change here.

This PR did not receive any negative feedback in any case.

**Comment by the-orange-cow** [comment]
> So this PR'd be aimed at particular, persistent friends who team up on every occasion?
> I think you are oversensitive on that problem

I was making a joke lmao

**Pull request merged**